### ISSUE TITLE
Assembly mismatch!

### ISSUE THREAD

**Issue by deokershesh** [dotnet/runtime#1207]
I have built Informix.Net.Core.dll (from System.Data.Odbc.dll by making appropriate changes) successfully and entire .NET Core 3.1 (from 'git clone https://github.com/dotnet/runtime.git'). While trying to create simple sample program referencing "Informix.Net.Core", I get below set of warnings and errors. Not sure what I am missing, any help would be appreciated.

C:\Program Files\dotnet\sdk\3.1.100\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Found conflicts between different versions of "System.Transactions.Local" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\shesh\DOTNET-Core-Informix-code\VS2019_projects\MyApp\MyApp.csproj]

CSC : error CS1705: Assembly 'Informix.Net.Core' with identity 'Informix.Net.Core, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' uses 'System.Data.Common, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' which has a higher version than referenced assembly 'System.Data.Common' with identity 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [D:\shesh\DOTNET-Core-Informix-code\VS2019_projects\MyApp\MyApp.csproj]

Program1.cs(23,38): error CS1705: Assembly 'Informix.Net.Core' with identity 'Informix.Net.Core, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' uses 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' which has a higher version than referenced assembly 'System.Runtime' with identity 'System.Runtime, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [D:\shesh\DOTNET-Core-Informix-code\VS2019_projects\MyApp\MyApp.csproj]

**Comment by mikedn** [comment]
> entire .NET Core 3.1 (from 'git clone https://github.com/dotnet/runtime.git').

This repository produces a .NET Core **5.0** build, if you want **3.1** you'll need to the appropriate branch in the old corefx repo, presumably https://github.com/dotnet/corefx/tree/release/3.1

**Comment by deokershesh** [comment]
I am confused, may be I got it wrong, I picked up entire "https://github.com/dotnet/runtime.git", built it and changed, System.Data.Odbc to Informix.Net.Core and created respective Informix.Net.Core.dll. I intend to use .NET Core 3.1 runtime. Should I get "https://github.com/dotnet/corefx/tree/release/3.1" for the same? Basically, I intend to create .NET Core 3.1 Provider/Driver for Informix database.

I already have the Informix Provider working with .NET Core 2.2, which I picked from "https://github.com/dotnet/corefx.git" may be few months back!
